Week3/homework/js-exercises/AddSix/app.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+const createBase = (base) => {
+    let count = 0;
+    return () => {
+        count++;
+        return base * count;
+    }
+}
+
+const addSix = createBase(6);
+console.log(addSix());
+console.log(addSix());
+console.log(addSix());
+console.log(addSix());
+
+// Δεν είμαι σίγουρος ότι καταλαβαίνω σωστά την εκφώνηση
+// από το να μην κάνω τίποτα λόγω αμφιβολίας έκανα αυτό
+// Νομίζω θα ήταν καλύτερα αν ζήταγε το παρακάτω...
```

Comment:
Please try to write everything in English. Even comments. You don't know who is going to read your code so it is better to be prepared